### PR TITLE
chore: update video akamai

### PR DESCRIPTION
**Pull request by aliandersonsn**
Search Nurture recommendations

# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/video/video-web/video-web-akamai/config.yml [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+  
+  ### Akamai player quickstart highlights
+  
+  The New Relic Akamai player quickstart has the following features
+  - Dashboards| Our dashboards proactively track metrics like video attempts, video plays, and average video start time. The dashboards also help you monitor other key indicators like connection buffering ratio, plays by app, and more.
+  - Alerts| You can get instant alerts on performance metrics like video playback failure, video start failure, and video start time.
+
+  ### How to monitor Akamai performance
+  
+  New Relic’s Akamai quickstart automatically instruments your Akamai player with our [video agent](https://developer.newrelic.com/instant-observability/video-web-akamai/794055cb-eeb3-4cca-b478-6cd1fb97678e/). It empowers you to monitor video applications with practical dashboards and alerts. The dashboard provides interactive visualizations to explore your data and understand context. In particular, the dashboard offers you the ability to drill down into performance details like the number of video attempts, number of video plays, and the overall average video start.
```
Comment (jpvajda) [inline quickstarts/video/video-web/video-web-akamai/config.yml:18]: Is this linking to itself? what's the reason to link the quickstart to it's own details page?

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: quickstarts/video/video-web/video-web-akamai/config.yml (hunk 1 above)
Comment (aliandersonsn) [reply quickstarts/video/video-web/video-web-akamai/config.yml:18]: @jpvajda no, that's incorrect. However, I don't love the links we do have. Let's remove the link for now and then I'll talk with Bradley about a better option.

Comment (jpvajda) [reply quickstarts/video/video-web/video-web-akamai/config.yml:18]:
```suggestion
  New Relic’s Akamai quickstart automatically instruments your Akamai player with our video agent. It empowers you to monitor video applications with practical dashboards and alerts. The dashboard provides interactive visualizations to explore your data and understand context. In particular, the dashboard offers you the ability to drill down into performance details like the number of video attempts, number of video plays, and the overall average video start.
```

**Review by jpvajda — CHANGES_REQUESTED** [review]
will merge once the url is removed

**Review by jpvajda — APPROVED** [review]

**Pull request merged**